homeassistant/components/enigma2/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "enigma2",
   "name": "Enigma2 (OpenWebif)",
-  "codeowners": ["@fbradyirl"],
+  "codeowners": ["@autinerd"],
```

Comment:
Is it intended that the previous code owner will be removed?

Reply:
As I understood in https://github.com/fbradyirl/openwebifpy/issues/17, he had no problems with me taking the project over. Maybe @fbradyirl can comment whether he wants to stay as code owner as well or I can take it over.

Reply:
Yes this is ok with me.